package.json

```diff
@@ -25,7 +25,9 @@
     "build": "tsc",
     "build-watch": "tsc --watch",
     "tslint": "tslint -c tslint.json 'src/plugin/*.ts'",
-    "test": "jest"
+    "test": "jest",
+    "test:android": "cordova-paramedic --verbose --platform android --plugin .",
+    "test:ios": "cordova-paramedic --verbose --platform ios --plugin ."
```

Comment:
should the emulators be specified here with `--target`?

Reply:
This is so annoying. I have tried and it looks like cordova-paramedic ignores the target arg and always picks the latest iphone simulator. Should I remove the pre boot of the simulator then?

Reply:
You can see here, how I ran directly the command and it doesn't pick it up https://app.circleci.com/pipelines/github/RevenueCat/cordova-plugin-purchases/263/workflows/8e03d50d-89ac-4b5b-bbfc-5944f8c7f39e/jobs/342

Reply:
what's the difference between this line and https://github.com/RevenueCat/cordova-plugin-purchases/pull/64/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R93?

Reply:
I'm confused, what do you mean it doesn't pick it up? in the logs you pointed to, it says: 

```
cordova-paramedic: running command cordova run ios --no-telemetry --no-update-notifier  --target "iPhone-11-Pro-Max, 13.3" --emulator
$ cordova run ios --no-telemetry --no-update-notifier  --target "iPhone-11-Pro-Max, 13.3" --emulator
cordova-paramedic: waiting for test results
Building for "iPhone 11 Pro Max" Simulator (com.apple.CoreSimulator.SimDeviceType.iPhone-11-Pro-Max, iPhone-11-Pro-Max).
Building project: /private/var/folders/pt/0xykrh9j62g34sc7znh47kbc0000gn/T/tmp-1158lk64KFCq3uGk/platforms/ios/HelloCordova.xcworkspace
	Configuration: Debug
	Platform: emulator
	Target: iPhone 11 Pro Max
```

Reply:
Sorry I didn't explain correctly. It doesn't pick up the `--target` parameter. 

As you can see in this commit https://github.com/RevenueCat/cordova-plugin-purchases/pull/64/commits/8d588a6691bd05cb72c59395eec6fac02453e689, I am passing `--target:"iPhone-11-Pro, 13.3"`, but looking at the logs in CircleCI https://app.circleci.com/pipelines/github/RevenueCat/cordova-plugin-purchases/263/workflows/8e03d50d-89ac-4b5b-bbfc-5944f8c7f39e/jobs/342 , it picked up `Target: iPhone 11 Pro Max`

It picked up the Max version. If you look at the top of the logs:

```
0. Paramedic config
   - platform: ios
   - action: run
   - args: 
   - plugins: .
   - useTunnel: false
   - verbose: true
   - cleanUpAfterRun: false
   - shouldUseSauce: false
 ```
 
The target is not part of the config either, so I think it's ignoring the `--target` completely. 


---

Answering to https://github.com/RevenueCat/cordova-plugin-purchases/pull/64#discussion_r540225165

There's no difference, I replaced the run command from `run test:ios` to directly calling cordova-paramedic in this commit https://github.com/RevenueCat/cordova-plugin-purchases/pull/64/commits/8d588a6691bd05cb72c59395eec6fac02453e689 because I wasn't sure the `--target` parameter was being passed to the `npm run test:ios`. I am going to revert the change now